spec/haskell/src/SEL4/Kernel/Thread/ARM.lhs

```diff
@@ -68,4 +68,9 @@ There is nothing special about idle thread activation on ARM.
 There is nothing special that needs to be done before calling nextDomain on ARM.
 
 > prepareNextDomain :: Kernel ()
-> prepareNextDomain = return ()
+> prepareNextDomain =
```

Comment:
comment above says "There is nothing special that needs to be done before calling nextDomain on ARM.", might want to tweak

Reply:
Changed to "When calling nextDomain on ARM, if CONFIG_ARM_HYPERVISOR_SUPPORT is set we first save and clear any current VCPU state, to ensure that we do not later read from or write to cross-domain state."